index.html

```diff
@@ -565,19 +677,31 @@ <h4>Script Type</h4>
                 <p>It can be adapted to produce a <a>Pre-Recording Script</a>,
                   and/or used as the basis for a further translation into the <a>Target Recording Language</a>.</p> 
                 <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Translation</a>.
-                  They MAY also contain <a>Original</a> language <a>Text</a>.</p>
+                  <a>Text Language Source</a> is set to the source language from which they were translated,
+                  and whose language is set to their actual language,
+                  indicating that they are <a>Translations</a>.
```

Comment:
Will reword, but not sure about having a defined term - what would the term be?

Reply:
It's not really the "definition" part that is important, but the link. It's not obvious how you "mark" something as Original or Translation.

Reply:
I'm resolving this conversation because I made a simplifying proposal in https://github.com/w3c/dapt/pull/179#discussion_r1428367490 which I think resolves it.